sync/src/blocks.rs

```diff
@@ -392,15 +392,15 @@ impl BlockCollection {
 						},
 						None => {
 							warn!("Got receipt with no header {}", h);
-							return Err(NetworkError::BadProtocol)
+							return Err(network::ErrorKind::BadProtocol.into())
```

Comment:
There is also `bail!` macro you could use:
`bail!(network::ErrorKind::BadProtocol)`

Reply:
I didn't know about it ;) I will use it next time, ok? :)

Reply:
Sure. I haven't yet decided which I like better, so not pushing it :)